streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -919,9 +911,7 @@ public void run() {
      * @throws IllegalArgumentException if {@code timeout} can't be represented as {@code long milliseconds}
      */
     public synchronized boolean close(final Duration timeout) throws IllegalArgumentException {
-        ApiUtils.validateMillisecondDuration(timeout, "timeout");
-
-        final long timeoutMs = timeout.toMillis();
+        final long timeoutMs = ApiUtils.validateMillisecondDuration(timeout, "timeout");
```

Comment:
I think `validateXXX` method shouldn't return value.
That's why I made it void in the first place.

Maybe we need a more convenient name for it?

Reply:
> That's why I made it void in the first place.

It's not `void` -- and this PR does not change it. I think it makes sense to let it return for this case though. To validate the given `Duration`, we need to call `#toMillis()` within `validateMillisecondDuration()` and thus, there is no reason to call it twice (even if this is not on the critical code path).

We can still rename it. Maybe something like, `getMillisecondDurationOrFailIfInvalid` ?